app/assets/javascripts/components/overview/overview_stats.jsx

```diff
@@ -108,96 +108,67 @@ const OverviewStats = ({ course }) => {
+        'redirects created': statistics['redirects created'],
+        'restorations performed': statistics['restorations performed'],
+        'reverts performed': statistics['reverts performed'],
+        'no data': statistics['no data'] }
+    ];
+
+    statsArray.forEach((stat) => {
+      Object.keys(stat).forEach((key) => {
+        if (stat[key] === 0) {
+          delete stat[key];
+        }
+      });
+    });
+
+    wikidataStats = statsArray.map((stat) => {
+      if (Object.keys(stat).length === 1) {
+        const label = Object.keys(stat)[0].split(' ').join('_');
+        return <OverviewStat
+          key={label}
+          id={(Object.keys(stat)[0].split(' ').join('-'))}
+          className={valueClass(label)}
+          stat={Object.values(stat)[0]}
+          statMsg={I18n.t(`metrics.${label}`)}
+        />;
+      }
+      if (Object.keys(stat).length > 1) {
+        const infoStats = [];
+        for (let i = 1; i < Object.keys(stat).length; i += 1) {
+          infoStats.push([Object.values(stat)[i], I18n.t(`metrics.${Object.keys(stat)[i].split(' ').join('_')}`)]);
+        }
+        const label = Object.keys(stat)[0].split(' ').join('_');
+
+        return <OverviewStat
+          key={label}
+          id={Object.keys(stat)[0].split(' ').join('-')}
+          className={valueClass(label)}
+          stat={Object.values(stat)[0]}
+          statMsg={I18n.t(`metrics.${label}`)}
+          info={infoStats}
+          infoId={`${label}-info`}
+        />;
+      }
+      return <div />;
+      });
+    }
+
 
-    const itemsInfo = [[statistics['items cleared'], I18n.t('metrics.items_cleared')]];
-    const claimsInfo = [[statistics['claims changed'], I18n.t('metrics.claims_changed')],
-    [statistics['claims removed'], I18n.t('metrics.claims_removed')]];
-    const aliasesInfo = [[statistics['aliases changed'], I18n.t('metrics.aliases_changed')],
-    [statistics['aliases removed'], I18n.t('metrics.aliases_removed')]];
-    const descriptionsInfo = [[statistics['descriptions changed'], I18n.t('metrics.descriptions_changed')],
-    [statistics['descriptions removed'], I18n.t('metrics.descriptions_removed')]];
-    const interwikiLinksInfo = [[statistics['interwiki links removed'],
-    I18n.t('metrics.interwiki_links_removed')]];
-    const labelsInfo = [[statistics['labels changed'], I18n.t('metrics.labels_changed')],
-    [statistics['labels removed'], I18n.t('metrics.labels_removed')]];
-    const mergedInfo = [[statistics['merged to'], I18n.t('metrics.merged_to')]];
-    const otherUpdatesInfo = [[statistics['qualifiers added'], I18n.t('metrics.qualifiers_added')],
-    [statistics['redirects created'], I18n.t('metrics.redirects_created')],
-    [statistics['restorations performed'], I18n.t('metrics.restorations_performed')],
-    [statistics['reverts performed'], I18n.t('metrics.reverts_performed')],
-    [statistics['no data'], I18n.t('metrics.no_data')]];
-
-    wikidataStats = (
```

Comment:
The approach here is much easier to read and make sense of than `map` version. I would suggest pushing the code for whether or not to render a stat with value zero into the OverviewStat component. Maybe it could be controlled with another prop, like `renderZero`. This way, we can easily control which stats should still be shown even if the value is zero, and which ones to hide — for both the Wikidata stats and the prior stats like `refCount` and `uploadCount`.